backend/handler/igdb_handler.py

```diff
@@ -137,33 +154,31 @@ def _search_screenshots(self, rom_id: int) -> list:
         ]
 
     @check_twitch_token
-    def get_platform(self, p_slug: str):
-        paltforms = self._request(
+    def get_platform(self, slug: str) -> IGDBPlatformType:
+        platforms = self._request(
             self.platform_url,
-            data=f'fields id, name; where slug="{p_slug.lower()}";',
+            data=f'fields id, name; where slug="{slug.lower()}";',
         )
 
-        platform = pydash.get(paltforms, "[0]", None)
+        platform = pydash.get(platforms, "[0]", None)
         if not platform:
             return {
                 "igdb_id": "",
```

Comment:
What do you mean? I think this is just a whitespace change.

Reply:
Oh maybe I misstarget the line, I meant the IGDB I'd because I was with the idea of returning it as int, so it had more sense to return a None instead of a empty string, but if we are going to stick with strings then the empty string is fine